docs/features/index.rst

```diff
@@ -13,7 +13,6 @@ Pootle FS
    :maxdepth: 2
 
    pootle_fs
-   pootle_fs_install_plugins
```

Comment:
Not sure it makes sense to move VCS plugins instructions to migration.  It might make sense when someone migrates from 2.8 -> 2.9.  But after that its a normal install situation, and someone won't be migrating.  It's a Pootle FS setup issue.

Reply:
Will have separate docs for "migrate to Pootle FS" and "integrate with VCS".